ocs_ci/deployment/deployment.py

```
@@ -3747,6 +3748,388 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+        try:
+            logger.info(
+                f"[ramenctl-e2e] Cloning ramenctl from {ramenctl_repo_url} "
+                f"(branch: {ramenctl_branch})"
+            )
+            try:
+                clone_repo(
+                    url=ramenctl_repo_url,
+                    location=ramenctl_clone_dir,
+                    branch=ramenctl_branch,
+                )
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)